base/applications/rapps/include/appview.h

```diff
@@ -201,7 +201,7 @@ class CAppInfoDisplay :
 };
 
 class CAppsListView :
-    public CUiWindow<CListView>
+    public CUiWindow<CWindowImpl<CAppsListView, CListView>>
```

Comment:
What's the reason of this change? Is it because CListView doesn't provide the direct way to modify the message map?

Reply:
CListView is a CWindow, which is just a wrapper for an HWND, meaning no message handling at all.